src/Pages/ChannelRequests.razor

```diff
@@ -78,9 +79,21 @@
                             </Addon>
                             <ValidationError/>
                         </Addons>
-                        <FieldHelp>@(_selectedDestNode == null ? "Node Not found" : $"Found Node: {_selectedDestNode.Name}") </FieldHelp>
-                    </Validation>
-                </EditTemplate>
+               <FieldHelp>
+                    @{
+                        if (_selectedDestNode == null)
+                        {
+                            @("Node Not found");
```

Comment:
You can use html directly here, no need for a razor code.

E.g.:

<p> Node not found </p>